### ISSUE TITLE
this.renderChunk Issue

### ISSUE THREAD

**Issue by sammygriffiths** [Bruno17/MIGX#211]
I want to use the renderChunk renderer so as to use the image+ snippet to display an image in the column. But I've found that using renderChunk sets whatever is in the renderChunk template as the value of the entry.

For a simple example; I have a migx TV where policies on animals can be added, I want the column to be rendered with h1 tags around the values, so I use the renderChunk renderer like so: [image](https://cloud.githubusercontent.com/assets/6638381/9765740/ac40b8be-570d-11e5-8169-96a34c70cd42.png).

I then edit the resource that this TV is applied to and add a value, it is rendered wrapped in h1 tags I would expect: [image](https://cloud.githubusercontent.com/assets/6638381/9765814/f9d73bac-570d-11e5-84ab-50575fd943e8.png).

If I then go to edit that value it has the h1 tags added to it's value, and if I save it it will be saved with these h1 tags and have more added by the renderer, and so on. [Image](https://cloud.githubusercontent.com/assets/6638381/9765870/36ae7ab8-570e-11e5-9ddb-171f1c81a4a4.png).

It's quite possible that I'm doing something wrong here, but surely this isn't the expected behaviour? Shouldn't the renderer only be responsible for how the value looks in the column and not affect the actual value?

Thanks.

**Comment by scorchsoft** [comment]
I wonder if this issue is related to the one I have just raised:
https://github.com/Bruno17/MIGX/issues/252

**Comment by Bruno17** [comment]
no, you allways have to use another fieldname for a renderchunk-column. There is no way arround at this time. The rendered value will be the value of that field in the record.

**Comment by scorchsoft** [comment]
Take a look at this page:
http://fluence.world/case-studies-and-success-stories 

We have the following feild list:
[example feild list.txt](https://github.com/Bruno17/MIGX/files/148535/example.feild.list.txt)

and the TPL for it looks like so:
[TPL look.txt](https://github.com/Bruno17/MIGX/files/148536/TPL.look.txt)

when the new field we added was "block_size"

You can see that when it loops the only HTML that it is outputting is the following:
```
<img src="/assets/components/phpthumbof/cache/JW%20–%20Cookham%20Wood.52c34b60b8700d8a1cf8241f05dd684616.jpg" alt="Our Work JW - Cookham Wood " class="img-responsive">`]]
 </div>
```
I.e. everything around it is gone
for reference I'm using MigX 2.9.7-pl Maybe you are on the latest nightly?
[migx-2.9.7-pl.transport.zip](https://github.com/Bruno17/MIGX/files/148542/migx-2.9.7-pl.transport.zip)

Here is what things look like in the back end:
![block values 01](https://cloud.githubusercontent.com/assets/10031309/13358197/ff7187ac-dca5-11e5-8cce-83c681648ab4.PNG)
![block values 02](https://cloud.githubusercontent.com/assets/10031309/13358200/00c9d406-dca6-11e5-8605-7c3257a5ec48.PNG)


I can send you the full website and database files if that helps?

**Comment by scorchsoft** [comment]
And just incase the page gets changed, here is an image of how it looks and the source code at the time:
![screencapture-fluence-world-case-studies-and-success-stories-1456504247181](https://cloud.githubusercontent.com/assets/10031309/13358273/5a2f9bde-dca6-11e5-9d8e-173604daf669.png)
[full page source.txt](https://github.com/Bruno17/MIGX/files/148565/full.page.source.txt)

**Comment by Bruno17** [comment]
maybe, you have too much nested output-filters.
what happens with:
           
```
            [[[[*inheritWorkFrom:isnot=``:then=`
                getImageList? 
                  &tvname=`OurWork`
                  &tpl=`tpl.workItem`
                  &docid=`[[*inheritWorkFrom]]`
            `:else=`
                getImageList? 
                  &tvname=`OurWork`
                  &tpl=`tpl.workItem`
                  &docid=`[[*id]]`
            `]]]]
```
